.github/scripts/build.sh

```diff
@@ -51,6 +52,9 @@ echo "===== sccache ====="
 ${DOCKER} sccache -s
 echo "===== build start ====="
 mkdir -p build
+# Ensure there's a $USER environment variable set with username corresponding to the uid of the user running the container
```

Comment:
Why?

Reply:
shoot. *just* pushed a rebased branch since you hadn't reviewed. bad timing.
Anyway, `cargo-generate` barfs if it can't get the user name.